Dev branch

butros10games opened the pull request:
Commit Type: Update

Updated the navbar to work with django

- Changed the link inport to use django static
- Removed the html body
- Removed body css

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

joosterws committed:
Project Name: [NatureTech]
Commit Type: Commit

Navbar aangemaakt

Functionele navbar gemaakt. Komen kleine aanpassingen aan.

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

joosterws committed:
Merge pull request #7 from butros10games/Desktop_Navbar

Project Name: [NatureTech]

butros10games committed:
Project Name: NatureTech
Commit Type: Update

Updated the navbar to work with django

- Changed the link inport to use django static
- Removed the html body
- Removed body css

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [Initial]

[Index pagina met styles erop (nog niet klaar)]

Signed-robleajub@gmail.com

AyubRoble committed:
Merge pull request #10 from butros10games/start_index_pagina

Initial commit index pagina

butros10games committed:
Project Name: NatureTech
Commit Type: Security

- Added how to work with the server in the docs
- Changed the settings file so the sec in a sepreat file sits.

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [update]

[Index pagina verbeterd, footer toegevoegd]

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

AyubRoble committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Docs

Added a forgotten command

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [Update]

code netjes gemaakt

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

AyubRoble committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

BobElsendoorn committed:
Project Name: NatureTech
Commit Type: Commit

Werkende boekingsbevestiging

Signed-off-by: Bob Elsendoorn <Bobelsendoorn@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Refactor

Changed the index structure to work with django

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: initial customer_general app

Added the customer_general app to the project

- Added the customer_general app
- Added the index routing
- Update the navbar to use the django templaing links

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Fix

Fixed a problem with the server_101.md

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Django isation

Changed index/nav page so it works with django

- Index img urls changed
- Index css url changed
- Navbar img url changed
- Navbar css url changed
- Changed img directory

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Bugfix

Fixed the error's that came up in the urls and views of customer_general

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

BobElsendoorn committed:
Project Name: NatureTech
Commit Type: Commit

Added booking model

Signed-off-by: Bob Elsendoorn <Bobelsendoorn@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: UUIDV7 + model

Added booking model

- Added the uuidv7 function
- Changed the booking_model

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Git update

Added settings.json to git ignore

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

joosterws committed:
Project Name: [NatureTech]
Commit Type: [Type of Commit]

[Short one-line description of the change]

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [Type of Commit]

[Short one-line description of the change]

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

AyubRoble committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [margin top]

top margin veranded

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

BobElsendoorn committed:
Project Name: NatureTech
Commit Type: Commit

Customer model added

Signed-off-by: Bob Elsendoorn <Bobelsendoorn@gmail.com>

BobElsendoorn committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [naam veranderd in index paget]

oase-weide

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

AyubRoble committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Update

Changed the index and navbar displays

- Changed the spacing and the placing of the navbar items
- Changed the color of the navbar to a darker green
- Changed the img on top to a background img
- Fliped the background img
- Added a darker filter over the img
- Added flexbox over the img to place the text on the right place

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [Woord veranderd]

oase naar weide

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [new file]

contact.css toegevoegd

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

joosterws reviewed (approved):
Gecontroleerd en ziet er goed uit.

butros10games committed:
Project Name: NatureTech
Commit Type: Enhancement

Added a admin pannel for the models

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [footer file gemaakt]

footer file met goede mappen constructie

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

AyubRoble committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

BobElsendoorn committed:
Project Name: NatureTech
Commit Type: commit
Added an database integration

Signed-off-by: Bob Elsendoorn <Bobelsendoorn@gmail.com>

BobElsendoorn committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Bugfix

Fixt a bug with the search_field with the customer admin pannel

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [Type of Commit]

[Short one-line description of the change]

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

AyubRoble committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

BobElsendoorn committed:
Project Name: NatureTech
Commit Type: Commit

Bug fixes + new features

Signed-off-by: Bob Elsendoorn <Bobelsendoorn@gmail.com>

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [Type of Commit]

[Short one-line description of the change]

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

AyubRoble committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: Contact formulier]

contact formulier toegevoegd

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

joosterws committed:
Project Name: [NatureTech]
Commit Type: Docs

Added info to the refresh commands

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

joosterws committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [Type of Commit]

contact pagina af, navbar updated

contact pagina af, navbar home wegehaald staat dubel want logo werkt ook

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

AyubRoble committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: update]

verstuur knop verbeterd
[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

joosterws committed:
Project Name: [NatureTech]
Commit Type: Feature

Added basic python scripts for BT scanning.

BT_scan.py for scanning and printing.
Bt_scna_upload for scanning and uploading to DB.

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [qolt]

width veranderd zodat de website niet zo breed is

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

AyubRoble committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

BobElsendoorn committed:
Project Name: NatureTech
Commit Type: commit

Added:
-confirmation email template
-changed the views

Signed-off-by: Bob Elsendoorn <Bobelsendoorn@gmail.com>

BobElsendoorn committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Update

Made the index page for the booking

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Enhancement

Made changed for rebust ness in the `booking_from` code

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

joosterws committed:
Project Name: [NatureTech]
Commit Type: Docs

Added basic commands to run and check some RPi services.

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

joosterws committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [Type of Commit]

[Short one-line description of the change]
foto veranderd

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

joosterws committed:
Project Name: [NatureTech]
Commit Type: Fix

Changing BT and BLE code

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

BobElsendoorn committed:
Project Name: NatureTech
Commit Type: Commit

Contactform backend

Signed-off-by: Bob Elsendoorn <bobelsendoorn@gmail.com>

BobElsendoorn committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

joosterws committed:
Project Name: [NatureTech]
Commit Type:  Fix

Made the BLE scan with RSSI, mac and name functional.

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

joosterws committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

AyubRoble committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Bufix

Fixed a problem with the csrf and the routing of the contact request

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

butros10games committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Enhancement

Added a ip logger for the bt reciever

- Ip logger model
- Ip logger view
- Ip display view

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

joosterws committed:
Project Name: [NatureTech]
Commit Type: Feature

Added the ability to send the RPi local IP to the server.

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

joosterws committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

BobElsendoorn committed:
Project Name: NatureTech
Commit Type:  commit

Cntactform backend

Signed-off-by: Bob Elsendoorn <bobelsendoorn@gmail.com>

BobElsendoorn committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

BobElsendoorn committed:
Project Name: NatureTech
Commit Type: [Type of Commit]

changed the user general view

Signed-off-by: Bob Elsendoorn <bobelsendoorn@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: merdge fix

removed a duplicat line

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Enhancement

Booking page layout, Maps container, agenda container

- Made the layout of the booking page.
- Made the agenda html.
- Changed the form html and css.
- Added a z-index to the navbar

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Bugfix

Fixed a number of bugs with the agenda display

- Line start direction fixed
- Agenda selection behavure fixed
- Fixed a visual bug with scroll;

Kown issue:
- The last day of the agenda is not selected in range

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Enhancement

Changed the number of people selection

- Changde the people selection field to a volwassen en kinderen button
- Added some missing information fields

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Enhancement

Added open en close dropdown tabs and plek form

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

joosterws committed:
Project Name: [NatureTech]
Commit Type: Feature

Send_ip now shows name and local IPv4 adress in termianl when run.

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

joosterws committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Enhancement

Made the code to the backend

- Data collection over alle forms
- Sending it to the backend
- Made the backend resonde with a json
- Added a Phone number save
- Changd the NumberofGuests to adults and children

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

joosterws committed:
Project Name: [NatureTech]
Commit Type: Fix

[Short one-line description of the change]

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Bugfix

Fixed a bug with the agenda date selection

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

joosterws committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Enhancement

Changed the slide behavure and view split

- Made the forms close when map slides open
- Added icons to the forms
- Split up the views for the booking page and the from send request

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Bugfix

A few small visual bugfixes

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Enhancement

Added a underlining on the navbar

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

joosterws committed:
Project Name: [NatureTech]
Commit Type: Fix

Made the script exit if it is unable to connect to the internet.

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

BobElsendoorn committed:
Project Name: NatureTech
Commit Type: Commit

Added:
*Confirmation mail for the admin
*Template for the confirmation mail to the customer
*Template for the confirmation mail to the admin.

Signed-off-by: Bob Elsendoorn <Bobelsendoorn@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Enhancement

Added a select marker on the navbar

- Added a hover animation
- Added some js to add active class to the navbar item of the page

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Enhacement

Added scroll lock to the booking page to prevent mis scroll

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Enhacement

Made some graphical changes

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Bugfix

Fixed the bug with scrolling on the fromContainer when body is not scrolled up

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

BobElsendoorn committed:
Project Name: NatureTech
Commit Type: commit

Added customer_general

Signed-off-by: Bob Elsendoorn <Bobelsendoorn@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Enhancement

Added a check for current date and gray out old dates

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

BobElsendoorn committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Enhancement

Added multi subdomain routing

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Enhancment

Changed the file directory structure to boer, boer-admin maps and auth system added

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

joosterws committed:
Project Name: [NatureTech]
Commit Type: Fix

Fixed startup script and added some basic info and future test script.

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

joosterws committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Remove

Removed the notes model

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

BobElsendoorn committed:
Project Name: NatureTech
Commit Type: Commit

Added the admin panel

Signed-off-by: Bob Elsendoorn <bobelsendoorn@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type:Enhancement

Made the orders page a sepreat page

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Folder changes

changed the file structure for the static files

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Directory changes

Changed all the static files to have the boer and boer-admin folders

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

BobElsendoorn committed:
Project Name: NatureTech
Commit Type: [Type of Commit]

[Short one-line description of the change]

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-off-by: Bob Elsendoorn <bobelsendoorn@gmail.com>

BobElsendoorn committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

joosterws committed:
Project Name: [NatureTech]
Commit Type: Feature

Script to test buttons/sensors like the door etc.

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

joosterws committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Bugfixes

Fixed some of the static imports

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

BobElsendoorn committed:
Project Name: NatureTech
Commit Type: Commit

Added styling for the bookingtable

Signed-off-by: Bob Elsendoorn <bobelsendoorn@gmail.com>

BobElsendoorn committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

joosterws committed:
Project Name: [NatureTech]
Commit Type: Feature

Detects movement high or low signal.

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

joosterws committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Enhacement

Changed the admin navbar a little

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

butros10games committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Enhacement

Added the comfigs for autoreload and tailwind

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Enhacement

Added the tailwind configs

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Enhacment

start navbar, suport taiwind configs and files

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: Tijd van lunch veranderd

14:00 ipv 2:00

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: Begin file nieuws en faciliteiten

[Short one-line description of the change]

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

butros10games committed:
Project Name: NatureTech
Commit Type: Enhacement fileWatcher

Added collectstatic command when a static file changes

Signed-off-by: ButrosGroot <butrosgroot@gmail.com>

butros10games committed:
Project Name: NatureTech
Commit Type: Enhacmenet Auth

Fixed some bugs with the auth system and made it inline with the requary ments in devops

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

butros10games committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

butros10games committed:
Project Name: NatureTech
Commit Type: Enhacement

Added login redirects when

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [Type of Commit]

[Short one-line description of the change]

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

AyubRoble committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [begin van de nieuws pagina

[Short one-line description of the change]

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

butros10games committed:
Project Name: NatureTech
Commit Type: Enhacement

Added urls and stared fixing the autorefresh

Signed-off-by: Butros Groot <Butrosgroot@gmail.com>

butros10games committed:
Merge branches 'Dev_branch' and 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: [Type of Commit]

[Short one-line description of the change]

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

AyubRoble committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

AyubRoble committed:
``` markdown
Project Name: [NatureTech]
Commit Type: laatste verse van de 2 pagina

2 pagina werkend, alleen inhoud toevoegen

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-robleajub@gmail.com

joosterws committed:
Project Name: [NatureTech]
Commit Type: Docs

Added a line for info,keywords, to look back on later.

[Optional longer description of the change, if needed]

[Additional context or notes, such as related issues or other details]

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>

joosterws committed:
Merge branch 'Dev_branch' of https://github.com/butros10games/NatureTech into Dev_branch

joosterws committed:
Project Name: [NatureTech]
Commit Type: Chore

Changed folder structure for test scripts

Signed-off-by: Joost Sluijmer <Joost.w.sluijmer@gmail.com>
